modules/apps/transfer/types/genesis.go

```diff
@@ -35,8 +35,5 @@ func (gs GenesisState) Validate() error {
 	if err := gs.DenomTraces.Validate(); err != nil {
 		return err
 	}
-	if err := gs.Params.Validate(); err != nil {
-		return err
-	}
```

Comment:
how come this is being removed?

Reply:
Transfer Params no longer have a `Validate` function. Every param in our case is a valid param since they are just booleans. Including a validate function would only reduce codecov since it's function signature would include an error. (The function itself would just say `return nit`)

Reply:
makes sense!

Reply:
The only reason a validate function existed before was that since the params were managed outside the transfer module, we needed to ensure `x/params` wasn't doing something funny by making sure the returned params were indeed booleans